src/bracket-double/double-elim.stories.jsx

```diff
@@ -78,3 +80,40 @@ export const DoubleEliminationNoParty = () => {
     </>
   );
 };
+
+export const DoubleEliminationDoubleFinalsLastGamesInLower = () => {
+  const [width, height] = useWindowSize();
+  const finalWidth = Math.max(width - 50, 500);
+  const finalHeight = Math.max(height - 100, 500);
+  return (
+    <>
+      <DoubleElimBracketLeaderboard
+        matches={lastGameInLowerMockData}
+        matchComponent={Match}
+        svgWrapper={({ children, ...props }) => (
+          <StyledSvgViewer width={finalWidth} height={finalHeight} {...props}>
+            {children}
+          </StyledSvgViewer>
+        )}
+      />
+    </>
+  );
+};
+export const DoubleEliminationDoubleFinalsLastGamesInUpper = () => {
+  const [width, height] = useWindowSize();
+  const finalWidth = Math.max(width - 50, 500);
+  const finalHeight = Math.max(height - 100, 500);
+  return (
+    <>
```

Comment:
Omit this? :)

Reply:
Removed unnecessary fragments alltogether from the project in the next commits